share/completions/dpkg-reconfigure.fish

```diff
@@ -8,7 +8,7 @@ complete -x -f -c dpkg-reconfigure -s h -l help     --description 'Display help'
 # General options
 complete -f -c dpkg-reconfigure -s f -l frontend -r -a "dialog readline noninteractive gnome kde editor web" --description 'Set configuration frontend'
 complete -f -c dpkg-reconfigure -s p -l priority -r -a "low medium high critical" --description 'Set priority threshold'
-complete -f -c dpkg-reconfigure -l default-priority --description 'Use default priority threshold'
+complete -f -c dpkg-reconfigure -l default-priority --description "Use current default ("(echo get debconf/priority | debconf-communicate | string match -r '\w*$')") priority threshold"
```

Comment:
>echo (echo "0 medium" | string split " ")[2]
works, although it looks a bit complex in this example.

That'll only work if it only ever prints one thing.

The current expression is almost fine for this, except it should not print blank lines probably - try `\w+$`. (Yes, this still relies on the fact that the matching is greedy i.e. `string match -r` always returns the longest possible match, but that's the more expected behavior anyway)

This way, it'll select the _last_ token.